superset/charts/commands/importers/v1/__init__.py

```diff
@@ -69,26 +70,29 @@ def _import(
                 database_ids[str(database.uuid)] = database.id
 
         # import datasets with the correct parent ref
-        dataset_info: Dict[str, Dict[str, Any]] = {}
+        datasets: Dict[str, SqlaTable] = {}
```

Comment:
While this is nowadays practically always `SqlaTable`, strictly speaking I think this should be `BaseDatasource`.

Reply:
Current importer doesn't support Druid datasource anyway, `BaseDatasource` does not have other attributes referenced at [L92](https://github.com/apache/superset/blob/f8270b0b80c51d32a23661d104416b6b5df395d5/superset/charts/commands/importers/v1/__init__.py#L92) and [L94](https://github.com/apache/superset/blob/f8270b0b80c51d32a23661d104416b6b5df395d5/superset/charts/commands/importers/v1/__init__.py#L94).